Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.6
+
+WORKDIR /
+COPY . /
+COPY app/test/test_no_password.pem /test
+COPY app/test/test_no_password.pub /test
```

Comment:
Should be at runtime, otherwise we tie the container to test @shahi645

Reply:
that is the purpose of the container, we'll see what we can do